gallery/src/components/NxVulnerabilityIndicatorLegend/NxVulnerabilityIndicatorLegendHorizontalExampleHalf.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxVulnerabilityIndicatorLegend } from '@sonatype/react-shared-components';
+
+const NxVulnerabilityIndicatorLegendHorizontalExample = () => (
+  <NxVulnerabilityIndicatorLegend header="Vulnerability Legend"
+                                  critical
+                                  high
+                                  medium
+    />
```

Comment:
One line

Reply:
> One line

And likewise in the other examples

Reply:
Fixed in: [2af62e0](https://github.com/sonatype/sonatype-react-shared-components/pull/1199/commits/2af62e0a71426434c93b12cf760ce6d59db8989b)